modules/update-restricted-network-cli.adoc

```diff
@@ -0,0 +1,129 @@
+    - <local_registry_host_name>:<local_registry_host_port>/<repository_name>/release
+    source: quay.io/openshift-release-dev/ocp-release
+  - mirrors:
+    - <local_registry_host_name>:<local_registry_host_port>/<repository_name>/release
+    source: quay.io/openshift-release-dev/ocp-v4.0-art-dev
+----
+<1> The `repositoryDigestMirrors` section must contain the `mirrors` data that
+is in the output of the `oc adm mirror release` command from when you mirrored
+the images for this release.
+
+. If the `repositoryDigestMirrors` section of the `ImageContentSourcePolicy`
+does not contain the `mirrors` for this release, add them to that section.
+
+. Start the cluster update:
++
+----
+$ oc adm upgrade --force --to-image <local_registry_host_name>:<local_registry_host_port>/ocp/release:<release_digest> <1>
+----
+<1> For `<local_registry_host_name>` and `<local_registry_host_port>`, specify
+the values that describe your mirror registry host name and port. For
+`<release_digest>`, specify the digest for the version that you want to upgrade
+to. You set this value as the `DIGEST` environment variable when you mirrored
+the content for the releas
+
+. Review the cluster version status history to monitor the status of the update.
+It might take some time for all the objects to finish updating.
++
+----
+$ oc get clusterversion -o json|jq ".items[0].status.history"
+
+[
+  {
+    "completionTime": null,
+    "image": "quay.io/openshift-release-dev/ocp-release@sha256:9c5f0df8b192a0d7b46cd5f6a4da2289c155fd5302dec7954f8f06c878160b8b",
+    "startedTime": "2019-06-19T20:30:50Z",
+    "state": "Partial",
+    "verified": true,
+    "version": "4.2.1"
+  },
+  {
+    "completionTime": "2019-06-19T20:30:50Z",
+    "image": "quay.io/openshift-release-dev/ocp-release@sha256:b8307ac0f3ec4ac86c3f3b52846425205022da52c16f56ec31cbe428501001d6",
+    "startedTime": "2019-06-19T17:38:10Z",
+    "state": "Completed",
+    "verified": false,
+    "version": "4.2.0"
+  }
+]
+----
++
+The history contains a list of the most recent versions applied to the cluster.
+This value is updated when the CVO applies an update. The list is ordered by
+date, where the newest update is first in the list. Updates in the history have
+state `Completed` if the rollout completed and `Partial` if the update failed
+or did not complete.
++
+[IMPORTANT]
+====
+If an upgrade fails, the cluster version reports a condition that explains the
+failure. Contact Red Hat support.
+====
+
+. After the update completes, you can confirm that the cluster version has
+updated to the new version:
++
+----
+$ oc get clusterversion
+
+NAME      VERSION     AVAILABLE   PROGRESSING   SINCE     STATUS
+version   4.2.1       True        False         2m        Cluster version is 4.2.1
+----
+
+. If you modified the contents of the `repositoryDigestMirrors` section of the
+`ImageContentSourcePolicy` and are no longer using the old mirrors, you can
+remove them:
++
+----
+$ oc edit ImageContentSourcePolicy
```

Comment:
If we are managing ImageContentSourcePolicy by creating a new object, there we should use `oc delete ImageContentSourcePolicy xxxx` to remove old ones.